### ISSUE TITLE
The framework can handle all labeled data？

### ISSUE THREAD

**Issue by tanjia123456** [scottjiao/Dynamic-selftraining-GCN#1]
hello, I want to use my own datasets, but unfortunately all data are labeled.
So, can I use this framework to handle all labeled data?

**Comment by scottjiao** [comment]
Yes it can handle all the dataset which normal GCN can handle. However, this framework fits well for few labels data, if the label information is too ample, I suggest to directly use  normal GCN as alternative.

**Comment by tanjia123456** [comment]
Actually, I haven't found a code that is suitable for all marked, many are unsupervised or semi-supervised. If you know, please tell me. In short, thank you for your reply

**Comment by scottjiao** [comment]
Graph neural networks are designed for semi-supervised problems with graph structures, especially for the node classification problems. If you have a strong need to deal with all-marked dataset, maybe traditional methods such as random-walk based methods, e.g. node2vec, or matrix factorization methods, e.g. pagerank or label propagation, are better.

**Comment by tanjia123456** [comment]
Sorry, my understanding is that I can use a semi-supervised framework and then add random walks. Is this correct?

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "scottjiao/Dynamic-selftraining-GCN"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月24日(星期四) 下午4:29
收件人:&nbsp;"scottjiao/Dynamic-selftraining-GCN"<Dynamic-selftraining-GCN@noreply.github.com&gt;;
抄送:&nbsp;"1248742467"<1248742467@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [scottjiao/Dynamic-selftraining-GCN] The framework can handle all labeled data？ (#1)





 
Graph neural networks are designed for semi-supervised problems with graph structures, especially for the node classification problems. If you have a strong need to deal with all-marked dataset, maybe traditional methods such as random-walk based methods, e.g. node2vec, or matrix factorization methods, e.g. pagerank or label propagation, are better.
